.babelrc

```diff
@@ -1,5 +1,5 @@
 {
   "presets": [
-    "es2015"
+    "es2015-native-modules"
```

Comment:
I've been using <https://www.npmjs.com/package/babel-preset-es2015-webpack> which looks to be about the same thing (slightly different implementations), but I'm going to use `es2015-native-modules` in my examples going forward, trusting that you know what works better with Webpack than I :)